docs/admin/settings/leo_gateway.md

```diff
@@ -131,6 +131,22 @@ And also, when size of a requested object more than `cache.cache_max_content_len
 
 ![Figure: Disk Cache Limits](../../assets/leofs-gateway-disk-cache-size.png)
 
+#### Configurations which may affect the cache behavior
+Whether or not an object is cached on LeoGateway is determined by below logics.
+
+- LeoGateway's cache feature is only available to small objects when cache.cache_disc_capacity = 0.
```

Comment:
LeoGateway's cache feature is only available to small objects when `cache.cache_disc_capacity = 0`.